detox/src/ios/expect.js

```diff
@@ -397,8 +397,8 @@ class WaitForElement extends WaitFor {
     //if ((!element instanceof Element)) throw new Error(`WaitForElement ctor argument must be a valid Element, got ${typeof element}`);
     this._element = element;
   }
-  toBeVisible() {
-    return new WaitForInteraction(this._invocationManager, this._element, new VisibleMatcher());
+  toBeVisible(percentage) {
```

Comment:
I think the default (i.e. `.75`) should be set here rather than in the platform-specific impl., such that it would by-definition be identical across all platforms. @noomorph wdyt?

Reply:
Hmm, I'm for adding a new matcher, first of all, `toBeVisibleBy` or something like that.